pinot-core/src/main/java/org/apache/pinot/core/data/table/TableResizer.java

```diff
@@ -79,14 +79,30 @@ public TableResizer(DataSchema dataSchema, QueryContext queryContext) {
     _numOrderByExpressions = orderByExpressions.size();
     _orderByValueExtractors = new OrderByValueExtractor[_numOrderByExpressions];
     Comparator[] comparators = new Comparator[_numOrderByExpressions];
+    int[] comparisonSign = new int[_numGroupByExpressions];
```

Comment:
Found that in Presto: "The default null ordering is NULLS LAST, **regardless of the ordering direction**". Will update the behviour in Pinot to match it.